jekyll/_cci2/language-ruby.md

```diff
@@ -247,7 +247,7 @@ steps:
 ```
 {% endraw %}
 
-Now we can setup our test database we'll use during the build.
+Now we can setup our test database we'll use during the build. We're using the `dockerize` [utility](https://github.com/jwilder/dockerize) to delay the [primary container]({{ site.baseurl }}/2.0/glossary/#primary-container)'s main process until the database service is available.
```

Comment:
to avoid using 'we' and for clarity change to -> The next section sets up the test database. It uses the `dockerize` [utility](https://github.com/jwilder/dockerize) to delay starting the main process of the  [primary container]({{ site.baseurl }}/2.0/glossary/#primary-container) until after the database service is available.

otherwise it looks good, thank you!

Reply:
Ha, the slow conversion from first-person plural is agonizing but delightful!